src/core/css/SearchResult.scss

```diff
@@ -1,13 +1,22 @@
+@import "~core/css/inc/vars";
+
+$border-radius: 9px;
+$secondary-color: #6a6a6a;
+
 .SearchResult {
-  border: solid #cfcfcf;
-  border-width: 1px 1px 0;
+  background: #f5f5f5;
+  border-bottom: 6px solid #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.5);
 
-  &:last-of-type {
-    border-bottom-width: 1px;
+  &:first-of-type {
+    border-top-left-radius: $border-radius;
+    border-top-right-radius: $border-radius;
   }
 
-  &:hover {
-    background-color: #efefef;
+  &:last-of-type {
+    border-bottom: 0;
+    border-bottom-left-radius: $border-radius;
```

Comment:
Yeah, it rounds the corners in the bottom, like in the mocks:

<img width="349" alt="screenshot 2016-09-28 23 46 11" src="https://cloud.githubusercontent.com/assets/90871/18935116/c97cc880-85d5-11e6-9142-689ce8187782.png">